compiler-rt/lib/rtsan/rtsan.cpp

```diff
@@ -43,10 +43,11 @@ static InitializationState GetInitializationState() {
       atomic_load(&rtsan_initialized, memory_order_acquire));
 }
 
-static auto PrintDiagnosticsAndDieAction(DiagnosticsInfo info) {
+static auto DefaultErrorAction(DiagnosticsInfo info) {
```

Comment:
I don't really like this name. This implies there are other `ErrorAction`s.

Reply:
Tried something different, let me know what you think or if you have another suggestion.

This new one matches the name of the parameter ExpectNotRealtime has